### PR TITLE
Implemented image trim function

### PR DESCRIPTION
**Pull request by kevich**

### CODE REVIEW

**Comment by kevich** [comment]
Seems like travis ended up support for php 5.3...
Do you agree to remove 5.3 from `.travis.yml`?

**Comment by kevich** [comment]
Fixed that by using differ dist for php 5.3

File: src/Barberry/Plugin/Imagemagick/Command.php [hunk 1]
```diff
@@ -41,6 +43,10 @@ public function configure($commandString)
             if (preg_match("@colorspace(Gray|CMYK|sRGB|Transparent|RGB)@", $val, $regs)) {
                 $this->colorspace = strlen($regs[1]) ? $regs[1] : null;
             }
+            if (preg_match("@trim((?:[0-F]{3,6})?)x((?:\d{1,2})?)@", $val, $regs)) {
```
Comment (Magomogo) [inline src/Barberry/Plugin/Imagemagick/Command.php]: Imagemagic allow to make `trim` without any parameters. May we support that?

Comment (Magomogo) [reply src/Barberry/Plugin/Imagemagick/Command.php]: Forget it, I've found you decide to command `trimx` to achieve that.

**Comment by Magomogo** [comment]
@kevich do you plan to implement something more, or should I release the new version?

**Comment by kevich** [comment]
@Magomogo in near future no. The function I implemented we already used in old image server, I just ported it to barberry since we're moving to barberry for now.

In nearest plans could be better support for animated gifs, since we're not really support resizing them for now. But that'll be I think next year.

So please release new version now.

**Pull request merged**